exp/api/v1beta1/gcpmanagedcontrolplane_types.go

```diff
@@ -142,6 +146,25 @@ type MasterAuthorizedNetworksConfigCidrBlock struct {
 	CidrBlock string `json:"cidr_block,omitempty"`
 }
 
+// NetworkPolicy represents configuration options for NetworkPolicy feature of the GKE cluster.
+type NetworkPolicy struct {
```

Comment:
If a user supplies a **NetworkPolicy** should at least one of these 2 fields be specified?